src/jrd/exe.cpp

```diff
@@ -258,6 +258,8 @@ static void execute_looper(thread_db*, Request*, jrd_tra*, const StmtNode*, Requ
 static void looper_seh(thread_db*, Request*, const StmtNode*);
 static void release_blobs(thread_db*, Request*);
 static void trigger_failure(thread_db*, Request*);
+static void savepointForget(thread_db* tdbb, Request* request, jrd_tra* transaction, SavNumber savNumber);
+static SavNumber savepointStart(Request* request, jrd_tra* transaction);
```

Comment:
I'd move them to an anonymous namespace, IMHO new functions should not use the old static approach.
And I always prefer naming `doSomething` rather than `somethingDo` -- i.e. `startSavepoint()` and `forgetSavepoint()`.